### ISSUE TITLE
WScript variable declaration in lib.d.ts is missing some members

### ISSUE THREAD

**Issue by zspitz** [Microsoft/TypeScript#2540]
The current declaration for the WScript object is missing some members:

```typescript
interface ITextReader {
    //incomplete
    Read(characters: number): string;
    ReadAll(): string;
    Close(): void;
}

declare var WScript: {
    Echo(s: any): void;
    StdErr: ITextWriter;
    StdOut: ITextWriter;
    Arguments: { length: number; Item(n: number): string; };
    ScriptFullName: string;
    Quit(exitCode?: number): number;

    //missing members:
    BuildVersion: number;
    FullName: string,
    Interactive: boolean;
    Name: string;
    Path: string;
    StdInL ITextReader;
    Version: string;
    ConnectObject(objEventSource: any, strPrefix: string): void;
    CreateObject(strProgID: string, strPrefix?: string): any;
    DisconnectObject(obj: any): void;
    GetObject(strPathname: string, strProgID?: string, strPrefix: string): any;
    Sleep(intTime: number): void;
}
```

**Comment by jasonwilliams200OK** [comment]
@zspitz, would be really awesome if you add language name after opening <code>```</code> for syntax highlighting, so it will look like:

```typescript
// used ```typescript
var blah = {
  a: 'A',
  b: function() { return undefined; }
};
```

Check out the list of supported languages by GitHub markdown parser: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml.

**Comment by zspitz** [comment]
@jasonwilliams200OK Done, thanks.

**Comment by mhegazy** [comment]
A PR would be appreciated. the sources for WScript typings are in [src\lib\scriptHost.d.ts](https://github.com/Microsoft/TypeScript/blob/master/src/lib/scriptHost.d.ts)

**Comment by mhegazy** [comment]
Fixed by #2560

**Comment by mhegazy** [comment]
thanks @zspitz !
